### PR TITLE
Forms Automated Test - LPS-134381 | master

### PR DESCRIPTION
**Pull request by lima-rafa**

### CODE REVIEW

**Comment by natocesarrego** [comment]
@lima-rafa, can you please rebase your code against to the latest branch master of Brian Chan's repo to solve the reported conflicts?

Thanks.

File: portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro [hunk 1]
```diff
@@ -319,6 +332,22 @@ definition {
 			value1 = "${conditionField}");
 	}
 
+	macro viewConditionNotPresent {
+		var key_conditionRowNumber = "${conditionRowNumber}";
+		var key_conditionField = "${conditionField}";
+		var key_conditionOperator = "${conditionOperator}";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro:338]: @lima-rafa, please sort alphabetically.

File: portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro [hunk 2]
```diff
@@ -355,6 +384,35 @@ definition {
 			value1 = "${conditionOperatorLowerCase}");
 	}
 
+	macro viewConditionPresent {
+		var key_conditionRowNumber = "${conditionRowNumber}";
+		var key_conditionField = "${conditionField}";
+		var key_conditionOperator = "${conditionOperator}";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro:390]: @lima-rafa, please sort alphabetically.

File: portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro [hunk 3]
```diff
@@ -355,6 +384,35 @@ definition {
...
+		}
+		else {
+			AssertElementPresent(locator1 = "FormRules#RULE_BUILDER_CONDITION_FIELD_PARAMETERS");
+		}
+	}
+
+	macro viewConditionSummaryNotPresent {
+		var key_ruleNumber = "${ruleNumber}";
+		var key_conditionField = "${conditionField}";
+		var key_conditionOperator = "${conditionOperator}";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro:406]: @lima-rafa, please sort alphabetically.

File: portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro [hunk 4]
```diff
@@ -355,6 +384,35 @@ definition {
...
+
+		if (isSet(conditionFieldConstant)) {
+			AssertElementNotPresent(locator1 = "FormRules#RULE_BUILDER_SUMMARY_CONDITION_EXTENDED");
+		}
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro:410]: @lima-rafa, apparently, this is not necessary for this test, right? If so, can you please remove this code and the path `FormRules#RULE_BUILDER_SUMMARY_CONDITION_EXTENDED`?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by lima-rafa** [comment]
@natocesarrego Alterado

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**